lib/ui/screens/event_screen.dart

```diff
@@ -118,6 +118,41 @@ class _EventScreenState extends State<EventScreen> {
   /// Create the title, start, end, location and price of an event.
   Widget _makeBasicEventInfo(Event event) {
     final textTheme = Theme.of(context).textTheme;
+
```

Comment:
This is quite big, it should probably be its own widget

Reply:
Done!